scss/_footer.scss

```diff
@@ -0,0 +1,30 @@
+footer {
+  background-color: $color-white-wrapper;
+  padding: 2rem 0;
+  .row {
+    justify-content: space-around;
+    .footer__privacy {
+      display: flex;
+      justify-content: center;
+
+      a {
+        color: $color-black;
+        text-decoration: none;
+        padding: 0 0.25rem;
+        span {
+        }
+      }
+    }
+    .footer__social-media {
+      display: flex;
+      justify-content: center;
+      span {
```

Comment:
Praktycznie nigdy nie stylujemy po tagach. Staraj się zawsze stylować po klasach.

Wyjątek to sytuacja, w której masz elementy jak `img` albo `svg`, którym nie zawsze nadaje się klasy.
Drugim wyjątkiem jest sytuacja, w której nie możesz ingerować w DOM i musisz stylować na strukturze, którą dostałeś